client/hack/scripts/generate-jar

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+JAR_DIR="$(dirname "${BASH_SOURCE[0]}")"
+GENERATOR_VERSION="6.0.0"
```

Comment:
This file isn't needed any more, but was helpful when testing different local `.jar` builds without needing to make a new container.

Reply:
I would prefer we avoid adding this file. A container-based and container-centric approach is preferable here.